Week03/lesson05/homework/week3-relaxr-part2/css/style.css

```diff
@@ -0,0 +1,223 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+ clear: both;
+}
```

Comment:
I also know I accidentally sent you an older version of the clearfix hack in some previous starter code. Use this version in future:

```
.clearfix:before, .clearfix:after {
  content: "";
  display: table;
}

.clearfix:after{
  clear: both;
}
```

Totally my fault, not yours! 😄